proto/regen/ecocredit/v1beta1/state.proto

```diff
@@ -93,6 +94,9 @@ message ProjectInfo {
 
     // metadata is any arbitrary metadata attached to the project.
     bytes metadata = 5;
+
+    // issuer is the address that is allowed to issue credits from this project
+    string issuer = 6;
```

Comment:
We actually don't want to do this restriction here and instead want any credit class issuer to be able to issue
```suggestion
```